src/controllers/index.js

```diff
@@ -29,6 +29,8 @@ router.use((req, res, next) => {
   /* istanbul ignore if */
   if (!platforms.includes(req.platform)) req.platform = 'pc';
 
+  if (req.header('Accept-Language') && req.header('Accept-Language').length) res?.set('Vary', 'Accept-Language');
```

Comment:
That's not really a good looking solution to check for ` req.header('Accept-Language').length > 0`

Reply:
i have to make sure the header exists first. this project (presently) doesn't have existential chaining coalescing at present, or i wouldn't have needed the first clause and could do `?.length`... and this is far more standard compared to `length > 0`, given that 0 is falsey already, so it's extraneous

Reply:
well the problem is that it relies on implicit type conversion basically it's harder to read, but my fault as i did not include the full line
```suggestion
  if (req.header('Accept-Language') && req.header('Accept-Language').length > 0) res?.set('Vary', 'Accept-Language');
```

Reply:
In my view implicit type conversion in js is evil, but if you find it more readable and understandable the way you have already done it then no harm in it.

Reply:
this might be just a rant about me not liking js type system (ts is no better in this case)

Reply:
yeah, i'm not doing this, cause this is how ts would want you to do it anyway

Reply:
i was wrong anyway, we definitely have nullish coalescing available, and if i added `> 0`, it'd still be type converting

Reply:
yeah but it would be explicit